README.md

```diff
@@ -31,7 +31,7 @@ identification in GPT-2 small](https://arxiv.org/abs/2211.00593):
 
 ## Setup
 
-Follow these steps to install the repo.  You'll first need python/pip, as well as node/npm.
+Follow these steps to install the repo.  You'll first need python/pip, as well as node/npm. Requires `Python >= 3.11`
```

Comment:
maybe merge with the second sentence?

Reply:
yeah sure

Reply:
The Python version requirement is already specified in `setup.py`. Is there a strong reason to specify it here as well?

Reply:
Yeah its just a heads up in the readme. I ran into error of requiring python >= 3.11 for the neuron viewer after I activated my env and ran `pip install -e .`

Reply:
Shall I close this @stevenbills ?

Reply:
Yes please.